### DISCUSSION TITLE
Unable to access internet after configurations.

### DISCUSSION THREAD

**Discussion by lty900301** [RaspAP/raspap-docker#42] [Q&A]
Hi. I configured the RaspAP using docker-compose. And the RaspAP docker is running with eth0 (though renamed to end0 with RaspAP for some reason) and can access internet. However the wlan0 doesn't connect to internet, and thus the hotspot client connect to internet either. None of the vpn or wireguard configured. 

<img width="994" alt="image" src="https://github.com/user-attachments/assets/90bd86e5-394c-4848-ad01-5275bd1e01ae">
<img width="961" alt="image" src="https://github.com/user-attachments/assets/5eaf47a0-96b9-4308-aabb-90f3d1d7561a">

**Comment by billz** [comment]
The renaming of eth0 to end0 is due to the adoption of predictable network interface names. This is done by the kernel, _not_ RaspAP. Start by consulting the docs on [configuring routing](https://docs.raspap.com/repeater/#configure-routing) for your interfaces.
